genapi.py

```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
 def setup(ns):
     """Ensure that we are ready to perform code generation. Returns typesystem."""
     # load raw table
-    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
+    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
```

Comment:
The .js file is input and the json file is the result of some processing
(and can be easily regenerated).

On Fri, Jun 5, 2015 at 1:03 PM, opotowsky <notifications@github.com> wrote:

> In genapi.py
> <https://github.com/cyclus/cymetric/pull/77#discussion_r31836270>:
>
> > @@ -908,12 +908,13 @@ def parse_args(argv):
> >  def setup(ns):
> >      """Ensure that we are ready to perform code generation. Returns typesystem."""
> >      # load raw table
> > -    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
> > +    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
>
> I guess I assumed that the .js and .json files should be together, but
> that was probably silly to assume! I'll update it to be created in the
> build directory.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cymetric/pull/77/files#r31836270>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192